imageproxy_test.go

```diff
@@ -299,20 +299,20 @@ func (t testTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	var raw string
 
 	switch req.URL.Path {
-	case "/ok":
+	case "/plain":
```

Comment:
with the behavior I suggested above (namely that an empty ContentTypes value is not treated special), then this change should be reverted, as well as the corresponding StatusForbidden below.